manifests/pipelines/s3-fetch-pipeline.yaml

```
@@ -204,7 +204,7 @@ spec:
                 app: $(params.model-name)-$(params.model-version)
             spec:
               containers:
-              - image: $(params.candidate-image-tag-reference)
+              - image: $(params.candidate-image-tag-reference)@$(tasks.build-container-image.results.IMAGE_DIGEST)
```

Shouldn't the imagePullPolicy be added here as well?

```suggestion
              - image: $(params.candidate-image-tag-reference)@$(tasks.build-container-image.results.IMAGE_DIGEST)
                imagePullPolicy: Always
```

---

Yep you are right, will add now